.circleci/config.yml

```diff
@@ -46,6 +46,8 @@ jobs:
           name: Run tests - google.cloud.bigtable
           command: |
             if [[ -n $(grep bigtable ~/target_packages) ]]; then
+              test_utils/scripts/circleci/prepare_bigtable.sh
+              export DO_NOT_DOWNLOAD_TEST_SERVER=0 
```

Comment:
Rather than making variables negative (which makes DO_NOT_DO_X=0 hard to interpret, as it's a double-negation), how about `export DOWNLOAD_BIGTABLE_TEST_SERVER=0` instead?

Reply:
See comment below.